scst-store/sca_scans/sca-scanning-results.md

```diff
@@ -1,4 +1,35 @@
 # SCA scanning results
+## <a id='110'></a>1.1.0
+
+### <a id='grype-sr'></a>Grype
```

Comment:
Are we no longer showing scan results for Black Duck?

Reply:
Black duck scan is in another story [META-513](https://jira.eng.vmware.com/browse/META-513)

Reply:
The black duck scans will be added in a follow up PR.

Reply:
It's a separate story, I have a commit for it.